app/views/shared/_header_nav.haml

```diff
@@ -20,8 +20,20 @@
           %li{:class => params[:controller] == "patients" && "active"}
             = link_to "Patients", patients_path
         - if has_access_to_samples_index?
-          %li{:class => ["samples", "batches"].include?(params[:controller]) && "active"}
-            = link_to "Samples", samples_path
+          %li{:class => "samples-dropdown"}
+            %span{:class => "samples-dropdown-text"}= "SAMPLES"
+            %span{:class => "samples-dropdown-arrow"}
+            %ul{:class => "samples-dropdown-ul"}
+              %li{:class => params[:controller] == "samples" && "active"}
+                = link_to "Individual samples" , samples_path
+                %span{:class => "check"}
+              %li{:class => params[:controller] == "batches" && "active"}
+                = link_to "Batches", batches_path
+                %span{:class => "check"}
+              %li{:class => params[:controller] == "sample_transfers" && "active"}
+                = link_to "Transfers", sample_transfers_path
+                %span{:class => "check"}
+            %script= "$('.samples-dropdown-ul').css('margin-left', $('.samples-dropdown-text').position().left+'px')"
```

Comment:
I had to compute this property to locate the dropdown in its correct position, I'm not sure this is the correct way to do it.

Reply:
There is a problem here with the parent `li` element. The items are `display: inline` which prevents `position: relative` to work. But that's just wrong. The items are not inline elements. They are definitely block elements. So we should refactor the navbar styles to fix that.